src/uniprotkb/components/protein-data-views/ProteinOverviewView.tsx

```diff
@@ -50,7 +51,18 @@ export const ProteinOverview: FC<{
     },
   ];
 
-  return <InfoList infoData={infoListData} />;
+  const descriptionMetaHead = `${
+    geneNamesData?.length ? geneNamesData[0].geneName?.value : ''
+  } - ${proteinName} - ${organismData?.scientificName}`;
+
+  return (
+    <>
+      <Helmet>
+        <meta name="description" content={`${descriptionMetaHead}`} />
+      </Helmet>
```

Comment:
Shouldn't we use this as the page title as in the current website?

Reply:
Looking at the current title it often gets cut off, so I was thinking it would be best to have this in the description and that we should review it later. Happy to add the protein name or gene name instead of - UniProt though?

Reply:
Happy with the accession if it was on purpose 👍🏽 

I would probably append UniProtKB instead of UniProt, as this could be changed depending on the namespace.
`9606 - taxonomy` would be more informative than `9606 - UniProt`. Even if that wouldn't really happen in this subsite.